ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -459,6 +466,27 @@ public boolean getAnimateMeasurementChange() {
         return animateMeasurementChange;
     }
 
+    /**
+     * If characterListWraparound is enabled, it means that the ticker animator will always animate
+     * upwards in the character list and looping around once it hits the end.
+     *
+     * <p>For example, given the character list "0123456789" commonly used for numbers, if
+     * characterListWraparound is set to true, the animation from 7 to 9 will proceed as
+     * 7 -> 8 -> 9, and the animation from 9 to 2 will proceed as 9 -> 0 -> 1 -> 2.
+     *
+     * @param characterListWraparound whether or not to enable character list wraparound.
+     */
+    public void setCharacterListWraparound(boolean characterListWraparound) {
```

Comment:
the reason i asked about the behavior, is it seems to me like this should be the default behavior (from a ui perspective), so it seems odd to expose a flag for it.

Reply:
you are saying that wrap-around should be the default behavior? im not sure if having the ticker always animate in one direction is good haha. i feel like wraparound is a rare/specific use-case, basically when you are trying to build a timer or something.

Reply:
i misunderstood a bit, lemme back up.

i think ideal behavior would be animating up when doing so would cause a wraparound, but the total distance via wraparound is lower than the total distance via not wrapping around. for cases where we go from 9 -> 0, that's a distance of 1 via wraparound, but 8 via not wrapping around. we should just always do the shortest.

having a force flag seems odd, i'm not sure that folks would want an 'always up behavior'. i guess some apps might, but it seems pretty limited.

so, i think we should work towards doing the shortest distance (wraparound or not), and not put it behind a flag (it should just be the behavior of the lib).

Reply:
i see, yeah i like that.